doc/generate_module_rst.py

```diff
@@ -170,9 +170,11 @@ def seen_in_dispatchers(self, seen: Set[int]) -> None:
             module.seen_in_dispatchers(seen)
         for function in self.functions:
             if isinstance(function, DocumentableDispatcher):
-                impls = function.obj.funcs.values()
-                for impl in impls:
-                    seen.add(id(impl))
+                # See comment below (for classes) about aliases.
+                if function.name.endswith("." + function.obj.__name__):
```

Comment:
Shouldn't this be checking for aliases of the implementations `impl` rather than of the dispatcher `function`?

Reply:
I guess you could have aliases at both levels, dispatcher and implementations. Adding a check at the implementation level is more tricky since there is no existing Documentable* wrapper class for the implementations -> so there is no pathname to reference, only the function objects themselves. It seems to me that it is unlikely we will have aliases for implementations, since there is not a need for users to call them directly. What do you think?